stdlib/LinearAlgebra/src/bidiag.jl

```diff
@@ -21,6 +21,7 @@ function Bidiagonal{T}(dv::AbstractVector, ev::AbstractVector, uplo::Union{Symbo
                convert(AbstractVector{T}, ev)::AbstractVector{T},
                uplo)
 end
+Bidiagonal{T, V}(A::Bidiagonal) where {T, V<:AbstractVector{T}} = Bidiagonal(convert(V, A.dv), convert(V, A.ev), A.uplo)
```

Comment:
The `convert`s here should be `copy` I think? Or rather `convert(V, copy(A.dv))`.

Reply:
Ah, right if they `A.dv` is type `V` this would return the reference not a copy. Thanks.